roles/translator/tproxy-config-example.toml

```diff
@@ -23,10 +23,16 @@ min_supported_version = 2
 min_extranonce2_size = 8
 coinbase_reward_sat = 5_000_000_000
 
-# optional jn config, if set the tproxy start on JN mode
-# [jn_config]
-# jn_address = "127.0.0.1:34264"
-# tp_address = "127.0.0.1:8442"
+# JN config (optional), if set the tproxy starts on JN mode
```

Comment:
IMO jn_config should be disabled by default.

Reply:
Done.
Even if I think this should be prioritize, since it's the biggest enhancement of our MVP2. But we'll try to emphasize it in docs 👍